qcodes/instrument/parameter.py

```diff
@@ -1120,7 +1121,7 @@ def __init__(self, name: str,
                             " set_raw is an error.")
         elif not self.settable and set_cmd is not False:
             if set_cmd is None:
-                self.set_raw: Callable = lambda x: x
+                self.set_raw: Callable[..., Any] = lambda x: x
```

Comment:
this lambda does not actually conform to the type... :)
```suggestion
                self.set_raw: Callable[[ParamRawDataType, Any kwargs ...], None] = lambda x: x
```

Reply:
Why not? Your suggestion is not valid syntax. There is not as far as I know any good way to type something like a callable that takes on specific argument but the rest can be dynamic.

Reply:
that's sad. but ok, `Callable[...` it will be then